src/sidebar/components/Annotation/AnnotationBody.js

```diff
@@ -63,24 +84,53 @@ function AnnotationBody({ annotation, settings }) {
             textStyle={textStyle}
             markdown={text}
             textClass={{
-              AnnotationBody__text: true,
               'p-redacted-content': isHidden(annotation),
             }}
           />
         </Excerpt>
       )}
-      {isCollapsible && (
-        <div className="hyp-u-layout-row--justify-right">
-          <LabeledButton
-            expanded={!isCollapsed}
-            onClick={() => setIsCollapsed(!isCollapsed)}
-            title={`Toggle visibility of full annotation text: Show ${toggleText}`}
-          >
-            {toggleText}
-          </LabeledButton>
+      {(isCollapsible || showTagList) && (
+        <div className="flex flex-row gap-x-2">
+          <div className="grow">
+            {showTagList && (
+              <TagList>
+                {tags.map(tag => {
+                  return (
+                    <TagListItem
+                      key={tag}
+                      tag={tag}
+                      href={shouldLinkTags ? createTagSearchURL(tag) : null}
+                    />
+                  );
+                })}
+              </TagList>
+            )}
+          </div>
+          {isCollapsible && (
+            <div>
```

Comment:
What does this unstyled `<div>` do?

Reply:
It is a flex-positioned container for the `LabeledButton`. It will stretch to the full height of the flex row, so it's necessary (as things are styled now) to have this container and put a button into it. To wit, if you eliminate the container `div` here right now,  you'd get:

<img width="425" alt="image" src="https://user-images.githubusercontent.com/439947/158182818-2c266361-7a85-4bdf-b67e-9886bfc3d9b9.png">